draft-ietf-taps-transport-security.md

```diff
@@ -418,7 +419,9 @@ like TCP.
 
 Application Payload Security Protocols:
 
-- SRTP {{?RFC7201}}
+- DTLS when used as a handshake protocol for SRTP {{?RFC7850}}
+- ZRTP {{?RFC4571}}
```

Comment:
Should the reference here be RFC6189, as we use that reference for ZRTP above?

Reply:
RFC4571 describes the framing of RTP-related protocols over connection oriented transports. It maybe needs both references.